src/setupTest.js

```diff
@@ -1 +1,11 @@
 import 'babel-polyfill';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import axios from 'axios';
+import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+jest.mock('@edx/frontend-platform/auth');
```

Comment:
just out of curiosity (since I haven't seen one before), was this documented or did you need to ask around? I'd be delighted to get to check out that documentation. :)

Reply:
It wasn't documented, but I was trying to solve an error that had to do with an HTTP request while I was testing. I looked at what was done in frontend-app-publisher (I've been learning from their testing files because this is my first time testing React), and this line (plus some others) solved my issue.